components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -490,6 +496,13 @@ export default class PostList extends React.PureComponent {
             };
         }
 
+        if (this.props.atLatestPost && this.props.shouldStartFromBottomWhenUnread) {
```

Comment:
Why do we need to check `atLatestPost` here? Wouldn't checking `shouldStartFromBottomWhenUnread` be enough?

Reply:
Thanks. `shouldStartFromBottomWhenUnread` would be enough.